sample-metadata/README.adoc

```diff
@@ -135,8 +135,16 @@ NOTE: the order of the columns are important, **assay name** SHOULD we always be
 
 TIP: All the possible _label_ values can be seen in the in the PRIDE CV under the https://www.ebi.ac.uk/ols/ontologies/pride/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPRIDE_0000514&viewMode=All&siblings=false[Label] node.
 
-In the case that technical and/or biological replicates have been measured, this information is not sufficient anymore.
-To be able to trace a given quantitative value to the exact replicate one needs encode this information as part of the experimental design.
+The "comment" columns in *SDRF* are included as a basic extensibility mechanism for local implementations. The name associated with the comment is included in square brackets in the column heading, and the value(s) entered in the body of the column. _comment_ columns could be used in various ways - to provide references to external files like raw files, or to include identifiers of objects in external systems.
+
+[[technical-biological-replicates]]
+=== Sample technical and biological replicates
+
+Technical and biological replicates are important to understand the experimental design. Multiple definitions can be attributed to technical and biological replicates and it will depend on the researcher, annotator to define and consider what in the data analysis is each concept. A good starting point for the SDRF specification is the following:
```

Comment:
I cannot disagree more with this statement: "Technical and biological replicates are important to understand the experimental design.". As a matter of fact, the rest of the paragraph states the opposite, attempting to describe your experiment in terms of replicates is subjective and relative to a given model.

Proposed alternative:
```
Different measurements are often categorized as (i) _Technical_ or (ii) _Biological_ replicates, based on whether they are (i) matched on all variables, e.g. same sample and same protocol; or (ii) different samples matched on explanatory variable(s), e.g. different patients receiving a placebo in a placebo vs. drug trial. Technical and biological replicates have different levels of independence, which must be taken into account during data interpretation.
For a given experiment, there are different levels to which samples can be matched - e.g. same sample, same sample same protocol, same sample same protocol same covariates - the definition of technical replicate can therefore vary based on the number of variables included. In addition, an experiment might be used in multiple models with different explanatory variable(s), and biological replicates in one model would not be replicates in another. Therefore, _Technical vs. Biological_ considerations, while sometimes relevant to analytical and statistical interpretation, fall beyond the scope of the sdrf format. However, data providers are encouraged to provide any identifier - e.g. _Biological_replicate_1_, _Technical_replicate_2_ - that would help linking the samples to their analytical and statistical analysis as comments.
```

Reply:
Thanks @mvaudel. I change the paragraph.